tests/data/src/pep518-3.0/setup.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
```

Comment:
I'll remove the churn and put it into (yet) another PR. :)

Regarding the style changes I'll make a comment about what they are in a moment.

> a debate for a PR specific to style rule changes

Which would be this PR.